src/digitalmarketplace/components/option-select/option-select.yaml

```diff
@@ -0,0 +1,96 @@
+params:
+- name: title
+  type: string
+  required: true
+  description: The title of the option select
+- name: name
+  type: string
+  required: true
+  description: The name of the option select element
+- name: options_container_id
+  type: string
+  required: true
+  description: The ID of the container for the Checkboxes component
+- name: items
+  type: array
+  required: true
+  description: The checkbox items
```

Comment:
Can we document what an item looks like?

Reply:
(We could probably just put a link to checkboxes item docs https://design-system.service.gov.uk/components/checkboxes/#options-checkboxes-example--items).